### DISCUSSION TITLE
Penumbra mods suddenly stopped working over default outfits

### DISCUSSION THREAD

**Discussion by lettey123** [xivdev/Penumbra#195] [Q&A] [answered]
The mods and everything were working fine for me. I was in an inn and everything was working perfectly. I then left the inn and went somewhere else to take a photo and suddenly none of the mods were working. When I have them enabled, they used to show up in previews and when I would change gear sets but now when I change my gear or hair-they come up as the game intended instead of the mods. I usually would have a history in everything I would touch and change in the "Effective Changes" tab but nothing shows up there anymore. I have tried everything, restarting my computer, the game, disabling Penumbra and re-enabling it...gone back to sanctuaries and even using Anamnesis to see if I can get it to work there too but nothing works. This was sudden! It was working fine and now none of my mods work or show anything but the default outfit they are meant to work over with. I have reloaded the files and reset the default settings but nothing is working. It is almost as if the plugin is not responding to the game's files and I have no idea how to fix this. I thought there would be a file to fix it in the downloads page but I cannot understand what they mean with the little amount of information they give so if my problem is something 'fixable' by another update or file, PLEASE tell me! Because it makes ZERO sense that it suddenly just stopped working! HELP!
![Screenshot 2022-04-25 212321](https://user-images.githubusercontent.com/104405738/165206904-0c77aabe-dced-4810-a48a-5ffdee34f240.jpg)
As you can see here, I have the peach dress over the faerie and it has always worked, but now it just comes up as the default and my hair too! Everything is default as if I have no mods all of a sudden but Penumbra still has everything. What do I do? Is this a setting where I have made Penumbra unresponsive to the game's files???????

**Comment by Ottermandias** [comment]
Show your Collections Tab and Settings.

**Comment by lettey123** [comment]
![image](https://user-images.githubusercontent.com/104405738/165412609-5e8edb1e-45af-4056-9101-c8d18dc5f283.png)

![image](https://user-images.githubusercontent.com/104405738/165412637-ba6a2bd0-ce21-4ee2-af16-2bb93bddefa9.png)
I also have switched off all settings and such but I have had it on these setting the whole time even when it was working fine.

**Comment by Ottermandias** [comment, marked as answer]
Have you noticed how there is a "None" besides Default Collection?
That means that there is no collection that is being applied by default. You probably want to change that to 'Default'.

**Reply by lettey123** [reply]
You fixed everything for me :) Thank you! It worked!
